spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -1,369 +1,372 @@
+        * [ ] Ensure `name` format matches "XXX-X Offboarding"
+        * [ ] Ensure `target` matches `DssExecLib.spotter()` address
+        * [ ] Ensure `ilk` format matches collateral type (`ilk`) name (`"XXX-X"`)
+        * [ ] Ensure `what` matches string `"mat"`
+        * [ ] Ensure `startTime` matches `block.timestamp`
+        * [ ] Ensure `start` uses variable `CURRENT_XXX_A_MAT`
+        * [ ] Ensure `start` matches current `spotter.ilk.mat` value
+        * [ ] Ensure `end` uses variable `TARGET_XXX_A_MAT`
+        * [ ] Ensure `end` value matches the instruction
+        * [ ] Ensure `end` allows liquidation of all remaining vaults (`end` is bigger than `collateral_type_collateralization_ratio * risk_multiplier_factor`)
+        * [ ] Ensure `duration` matches the instruction
+    * [ ] Spotter price is updated via [`DssExecLib.updateCollateralPrice(ilk)`](https://github.com/makerdao/dss-exec-lib/blob/v0.0.9/src/DssExecLib.sol#L374) IF collateral have no running oracle (i.e. relevant `PIP_` contract have outdated `zzz` value)
+    * [ ] Spotter price is updated after all other actions
+    * [ ] Offboarding is tested at least via [`_checkIlkClipper` helper](https://github.com/makerdao/spells-mainnet/blob/7400e91c4f211fc24bd4d3a95a86416afc4df9d1/src/DssSpell.t.base.sol#L856)
+* IF RWA updates are present
+  * IF `doc` is updated
+    * [ ] [`_updateDoc` helper](https://github.com/makerdao/spells-mainnet/blob/7400e91c4f211fc24bd4d3a95a86416afc4df9d1/archive/2023-09-27-DssSpell/DssSpell.sol#L76-L87) is copied one-to-one from the archive and defined above `actions`
+    * [ ] `_updateDoc(ilk, doc)` is called in the spell
+  * IF debt ceiling is updated
+    * IF AutoLine update is requested by the Exec Doc
+      * [ ] Parameters are set via [`DssExecLib.setIlkAutoLineParameters(ilk, amount, gap, ttl)`](https://github.com/makerdao/dss-exec-lib/blob/v0.0.9/src/DssExecLib.sol#L648) or [`DssExecLib.setIlkAutoLineDebtCeiling(ilk, amount)`](https://github.com/makerdao/dss-exec-lib/blob/v0.0.9/src/DssExecLib.sol#L658)
+    * IF regular debt ceiling (`vat.ilk.line`) update is requested by the Exec Doc
+      * [ ] Collateral type (`ilk`) have [`AutoLine`](https://github.com/makerdao/dss-auto-line/tree/master) disabled previously or in the spell
+      * [ ] Debt ceiling (`vat.ilk.line`) is updated, via either:
+          * [`DssExecLib.increaseIlkDebtCeiling(ilk, amount, global)`](https://github.com/makerdao/dss-exec-lib/blob/v0.0.9/src/DssExecLib.sol#L621C14-L621C36)
+          * [`DssExecLib.decreaseIlkDebtCeiling(ilk, amount, global)`](https://github.com/makerdao/dss-exec-lib/blob/v0.0.9/src/DssExecLib.sol#L634)
+          * [`DssExecLib.setIlkDebtCeiling(ilk, amount)`](https://github.com/makerdao/dss-exec-lib/blob/v0.0.9/src/DssExecLib.sol#L611)
```

Comment:
```suggestion
      * Debt ceiling (`vat.ilk.line`) is updated, via EITHER:
          * [ ] [`DssExecLib.increaseIlkDebtCeiling(ilk, amount, global)`](https://github.com/makerdao/dss-exec-lib/blob/v0.0.9/src/DssExecLib.sol#L621C14-L621C36)
          * [ ] [`DssExecLib.decreaseIlkDebtCeiling(ilk, amount, global)`](https://github.com/makerdao/dss-exec-lib/blob/v0.0.9/src/DssExecLib.sol#L634)
          * [ ] [`DssExecLib.setIlkDebtCeiling(ilk, amount)`](https://github.com/makerdao/dss-exec-lib/blob/v0.0.9/src/DssExecLib.sol#L611)
```

Reply:
Is it the same as above? ([regarding a single checkbox per check](https://github.com/makerdao/pe-checklists/pull/18#discussion_r1461968699))